.babelrc

```diff
@@ -4,13 +4,12 @@
       "@babel/preset-env",
       {
 		"targets": {
-		  "node": "current"
+		  "browsers": "last 2 version, > 1%, not dead, ie >= 10"
```

Comment:
@DevinWalker @kevinwhoffman I added support for old browsers. Do you want to edit this?

Reply:
update `> 1%` to `> 0.05%`

Reply:
@ravinderk Remove `not dead` as discussed on call.

Reply:
https://github.com/impress-org/give/pull/4050/commits/e7ab84bd902b15ed93ac4ef91a4b41188a3568e1